README.md

```diff
@@ -13,11 +13,7 @@ features, test coverage, etc.
 
 As a data sink:
 ```js
-const NoFilter = require('nofilter')
-// In ES6:
-// import NoFilter from 'nofilter'
-// In typescript:
-// import NoFilter = require('nofilter')
+import NoFilter from 'nofilter'
```

Comment:
TypeDoc generates docs where the class name is `default` with this approach.  I'd prefer `import {NoFilter} from 'nofilter'`.  We can re-export it as the default as well, if desired.

Reply:
changed to named export